### PR TITLE
Add unit tests to helper

### PR DESCRIPTION
**Pull request by robwise**
A reconciled version of PR #43

### CODE REVIEW

File: spec/dummy/spec/helpers/react_on_rails_helper_spec.rb [hunk 1]
```diff
@@ -34,4 +34,40 @@
...
+      subject { react_component("App", {}, id: id) }
+
+      let(:id) { "shaka_div" }
+
+      it { is_expected.to include id }
+      it { is_expected.not_to include react_component_div }
+    end
+  end
+
+  describe "#server_render_js" do
```
Comment (robwise) [inline spec/dummy/spec/helpers/react_on_rails_helper_spec.rb:61]: These tests were originally calling just `#render_js` and got method missing, so I assumed they meant `#server_render_js`.

File: spec/dummy/spec/helpers/react_on_rails_helper_spec.rb [hunk 2]
```diff
@@ -34,4 +34,40 @@
...
+    subject { server_render_js("this.HelloString.world()") }
+
+    let(:hello_world) do
+      "    Hello WORLD! Will this work?? YES! Time to visit Maui\n    \n"
```
Comment (robwise) [inline spec/dummy/spec/helpers/react_on_rails_helper_spec.rb]: I had to change the expected message to include the whitespace at the beginning and end, as well as the 2nd `/n`. Is this desired behavior?

Comment (justin808) [inline spec/dummy/spec/helpers/react_on_rails_helper_spec.rb]: @robwise Good question. I'm not sure why there's extra white space. Might be worth a quick look. Also, the test could trim the subject, so as not to require the extra whitespace.

Comment (justin808) [inline spec/dummy/spec/helpers/react_on_rails_helper_spec.rb]: Let's definitely see where this whitespace is coming from.

**Comment by robwise** [comment]
@justin808 Okay, I think this should be good to go. Whitespace issue fixed in commit 8c04ed2. All of Mario's tests now pass!

**Comment by justin808** [comment]
LGTM. Thanks! Great job @mapreal19 and @robwise!

**Comment by mapreal19** [comment]
Thanks @robwise! :clap:

**Comment by robwise** [comment]
@mapreal19 you bet! nice job!
